README.md

```diff
@@ -136,7 +135,8 @@ image.
 For example, the [`ezxenstore`](https://github.com/xapi-project/ezxenstore)
 project can be built in the following way using the `xenserver/xs-opam:latest`
 image:
-```
+
+```bash
```

Comment:
Have you used a particular tool to find these problems, or was it by visual inspection?

Reply:
I develop on VSCode with this plugin https://github.com/DavidAnson/vscode-markdownlint.